src/app/group/group.controller.js

```diff
@@ -1,18 +1,29 @@
 'use strict';
 
 angular.module('studyBuddy')
-    .controller('GroupCtrl', function($firebaseObject, $firebaseArray){
-        var ref = new Firebase('https://study-buddy.firebaseio.com/groups');
-        this.obj = $firebaseArray(ref);
-        console.log("banana");
-        this.groupsObject = {};
+    .controller('GroupCtrl', function($firebaseObject, $firebaseArray, Auth){
+//    this.login = Auth.fbLogin;
+    this.logout = Auth.logout;
+    var self = this;
+    Auth.onAuth(function(user){
+        self.user = user;
         
-        this.newGroup = {
+    });
+//    var myGroup = self.user.child('groupInfo');
+//    console.log(myGroup);
+    var ref = new Firebase('https://study-buddy.firebaseio.com/users/' + self.user.$id);
+    var ref2 = $firebaseObject(new Firebase('https://study-buddy.firebaseio.com/users/' + self.user.$id))
+    ref2.$loaded()
+    .then(function(data){
+        self.name = data.name
+    })
+    this.obj = $firebaseArray(ref);
```

Comment:
Oh man, this was committed mid changes, oops! Yeah the last commit, i have the group info being submitted under the user. I was having trouble grabbing info from the facebook object. It looked as though i could just follow the trail into the object putting .facebook.displayName but was getting undefined on everything except the self.user. I will commit what the code should have looked like